### ISSUE TITLE
Lazy loading does not work as expected

### ISSUE THREAD

**Issue by smares** [nuxt-modules/i18n#1678]
Following the documentation on lazy loading, the following configuration

```js
...
i18n: {
  defaultLocale: 'de',
  detectBrowserLanguage: false,
  langDir: 'locales/',
  lazy: true,
  locales: [
    {
      code: 'de',
      file: 'de.json',
      iso: 'de-DE',
      name: 'Deutsch',
    },
    {
      code: 'en',
      file: 'en.json',
      iso: 'en-US',
      name: 'English',
    },
  ],
},
...
```

leads to errors running `npm run dev`

```
@nuxtjs/i18n Failed locale loading: Module "file:///c:/Users/sMares/Projekte/nuxt-app/locales/de.json" needs an import assertion of type "json"                                                                                                                                 17:27:00
[intlify] Not found 'welcome' key in 'de' locale messages. 
```

I then tried adding

```js
vueI18n: {
  fallbackLocale: 'de',
},
```

which solved the problem leading only to the following warning

```
(node:23188) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time                                                                                                                                                      17:27:40
```

However, this leads to the `de.json` (or `de.js` in production) chunk always being loaded regardless of Nuxt already injecting the translations into the components. So it's not lazy at all. 😄 

Am I doing something wrong?

**Comment by kazupon** [comment]
Thank you for your reporting!
Same #1518 
We will fix this issue until v8 official release.

Thanks

**Comment by smares** [comment]
#1518 is about the `needs an import assertion of type "json"` error, but if I ignore that and run `yarn build` and then serve the production build, `de.js` is loaded by `entry.js` despite the fact that the components already contain the German texts. Why is that?

It happens even if I leave out the block 

```
vueI18n: {
  fallbackLocale: 'de',
},
```

**Comment by kazupon** [comment]
I've just relelase v8.0.0-beta.7
please check it!
Thanks!

**Comment by smares** [comment]
Given my example, I have two locales `de` and `en` using lazy loading. Default locale is `de`. Fallback locale is either unset or `de` (tried with both scenarios).

`yarn dev` or `yarn build` (tried with both) then visit `localhost:3000`. The HTML is delivered to the browser with all translation keys already replaced with their German `de` strings. Is it normal that `de.json` (`yarn dev`) / `de.js` (`yarn build`) is requested by the browser?

![image](https://user-images.githubusercontent.com/818380/206742408-9e647e93-baca-44aa-8605-77d51a40ca53.png)

![image](https://user-images.githubusercontent.com/818380/206742754-5931222a-ab96-4a67-a01c-7b5da8275385.png)

My use-case is identical to #1701 and I was hoping to achieve that using lazy loading in order to minimize the amount of data sent to the user's browser.

**Comment by angelhdzmultimedia** [comment]
Experiencing this.

 I have locales `en` and `es`.
 
 When I set `defaultLocale` to `es`, it loads, but then the `en` doesn't load, `$t('welcome') `prints the literal `'welcome'` string, and the browser console throws `[intlify] Not found 'welcome' key in 'en' locale messages. `
 
 So after playing with different properties, I added `vueI18n: { fallbackLocale: "en" }` and now both languages load and error goes away. But if I add  `vueI18n: { fallbackLocale: "es" }`, `en` won't load. I think I can solve this by adding an array as per the docs, with both langs as fallback.
